resources/dtd/database.dtd

```diff
@@ -126,8 +126,8 @@ PHP class or method name.
   phpName CDATA #IMPLIED
   refPhpName CDATA #IMPLIED
   interface CDATA #IMPLIED
-  onUpdate (CASCADE|cascade|SETNULL|setnull|RESTRICT|restrict|NONE|none) "none"
-  onDelete (CASCADE|cascade|SETNULL|setnull|RESTRICT|restrict|NONE|none) "none"
+  onUpdate (CASCADE|cascade|SETNULL|setnull|RESTRICT|restrict|NO ACTION|NONE|none) "none"
+  onDelete (CASCADE|cascade|SETNULL|setnull|RESTRICT|restrict|NO_ACTION|NONE|none) "none"
```

Comment:
Should this be NOACTION in light of SETNULL etc? To avoid the spacing?

Reply:
good catch, the DTD validation doesn't play nice with spaces (probably needs a `%20` there for the space? I added an override for 'NOACTION'.